content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
+:guilabel:`Keep Alternatives`.
 
-:guilabel:`Cancel Alternatives` automatically cancels the alternative purchase orders.
-:guilabel:`Keep Alternatives` keeps the alternative purchase orders open, so they can still be
-accessed if any additional product quantities need to be ordered. Once all products are ordered,
-:guilabel:`Cancel Alternatives` can be selected from whichever :abbr:`PO (Purchase Order)` is open.
+If this |PO| should **not** be confirmed, click :guilabel:`Discard`.
 
-To view a detailed form of one of the :abbr:`RFQs (Requests for Quotation)` listed, click the line
-item for that quotation. This causes an :guilabel:`Open: Alternative POs` pop-up window to appear,
-from which all details of that particular quotation can be viewed. Click :guilabel:`Close` when
-finished.
+Selecting :guilabel:`Cancel Alternatives` automatically cancels the alternative |RfQs|. Selecting
+:guilabel:`Keep Alternatives` keeps the alternative |RfQs| open, so they can still be accessed if
+any additional product quantities need to be ordered later.
 
-.. image:: calls_for_tenders/calls-for-tenders-keep-cancel-alternatives.png
+Once all products are ordered, :guilabel:`Cancel Alternatives` can be selected from whichever |PO|
+is open at that time.
+
+.. image:: calls_for_tenders/calls-for-tenders-keep-or-cancel.png
    :align: center
    :alt: Keep or cancel pop-up for alternative RFQs.
 
-From the :guilabel:`Alternative Warning` pop-up window, click :guilabel:`Keep Alternatives` to keep
-all alternative quotations open for now. Then, click :guilabel:`Requests for Quotation` (in the
-breadcrumbs, at the top of the page) to navigate back to an overview of all :abbr:`RFQs (Requests
-for Quotation)`.
+To view a detailed form of one of the |RfQs| listed, click the line item for that quotation. This
+opens an :guilabel:`Open: Alternative POs` pop-up window, from which all details of that particular
+|RfQ| can be viewed.
+
+Once ready, click :guilabel:`Close` to close the pop-up window.
+
+If all alternative |RfQs| should stay open, from the :guilabel:`What about the alternative Requests
+for Quotations?` pop-up window, click :guilabel:`Keep Alternatives`.
+
+Then, click :guilabel:`Requests for Quotation` (in the breadcrumbs, at the top of the page) to
+navigate back to an overview of all |RfQs|.
+
+Click into the remaining |RfQs| that contain products that need to be ordered, and click
+:guilabel:`Confirm Order`.
 
-Click into the remaining quotation(s) that contain products that need to be ordered, and click
-:guilabel:`Confirm Order`. This causes the :guilabel:`Alternative Warning` pop-up window to appear
-again. This time, click :guilabel:`Cancel Alternatives` to cancel all other alternative :abbr:`RFQs`
-linked with this quotation.
+This opens the :guilabel:`What about the alternative Requests for Quotations?` pop-up window. This
+time, click :guilabel:`Cancel Alternatives` to cancel all other alternative |RfQs| linked with this
+order.
```

Comment:
What's the purpose behind this? A little clarification on why they would need to do this could be helpful.

Reply:
Clarified the reasoning behind it in this sentence. Suggestion implemented ✅